plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/KVMPhysicalDisk.java

```diff
@@ -81,7 +83,9 @@ public KVMPhysicalDisk(String path, String name, KVMStoragePool pool) {
 
     @Override
     public String toString() {
-        return "KVMPhysicalDisk [path=" + path + ", name=" + name + ", pool=" + pool + ", format=" + format + ", size=" + size + ", virtualSize=" + virtualSize + "]";
+        return "KVMPhysicalDisk [path=" + path + ", name=" + name + ", pool=" + pool +
```

Comment:
is this format sacret, or can we use ReflectionUtils for the `toString()` implementation?

Reply:
:shrug: I was just playing along :)

Reply:
yeah, i see, which is fine, but it seems it is growing out of hand and ReflectionUtils can cleanup the implementation a bit (no blocker)

Reply:
changed it